app/uk/gov/hmrc/individualincomedesstub/controller/DocumentationController.scala

```diff
@@ -16,19 +16,22 @@
 
 package uk.gov.hmrc.individualincomedesstub.controller
 
+import akka.stream.Materializer
 import controllers.Assets
-import play.api.Configuration
+import play.api.{Configuration, Logging}
 import play.api.mvc.{Action, AnyContent, ControllerComponents}
+import play.filters.cors.CORSActionBuilder
 import uk.gov.hmrc.individualincomedesstub.views._
 import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 
 import javax.inject.{Inject, Singleton}
+import scala.concurrent.ExecutionContext
 
 @Singleton
-class DocumentationController @Inject()(configuration: Configuration,
-                                        cc: ControllerComponents,
-                                        assets: Assets)
-    extends BackendController(cc) {
+class DocumentationController @Inject()(configuration: Configuration, cc: ControllerComponents, assets: Assets)(
+  implicit materializer: Materializer,
+  executionContext: ExecutionContext)
+    extends BackendController(cc) with Logging {
```

Comment:
It doesn't look like logging is used here?

Reply:
Ah, sorry, well spotted, left over by mistake, correcting now